model/BrainPad.cpp

```diff
@@ -44,7 +44,9 @@ BrainPad::BrainPad() :
     buttonUp(io.buttonUp, ID_PIN_BUTTON_UP, DEVICE_BUTTON_ALL_EVENTS, ACTIVE_LOW, PullMode::Up),
     buttonDown(io.buttonDown, ID_PIN_BUTTON_DOWN, DEVICE_BUTTON_ALL_EVENTS, ACTIVE_LOW, PullMode::Up),
     buttonLeft(io.buttonLeft, ID_PIN_BUTTON_LEFT, DEVICE_BUTTON_ALL_EVENTS, ACTIVE_LOW, PullMode::Up),
-    buttonRight(io.buttonRight, ID_PIN_BUTTON_RIGHT, DEVICE_BUTTON_ALL_EVENTS, ACTIVE_LOW, PullMode::Up)
+    buttonRight(io.buttonRight, ID_PIN_BUTTON_RIGHT, DEVICE_BUTTON_ALL_EVENTS, ACTIVE_LOW, PullMode::Up),
+    space(CoordinateSystem::NORTH_EAST_UP, true, COORDINATE_SPACE_ROTATED_0),
```

Comment:
Do we need to store the coordinate system on the BrainPad? Can it be constructed in-place in the accelerometer initializer?